### DISCUSSION TITLE
CHART.JS 3 and plugin Zoom

### DISCUSSION THREAD

**Discussion by stockiNail** [chartjs/chartjs-plugin-zoom#460] [Q&A] [answered]
I would like to know when the plugin could be released in a stable version (no beta or rc) for Chart.js 3.x.
In some environments it's not allowed to use beta or release candidate.

**Comment by kurkle** [comment]
I think its really close to that quality. Will probably release later today.

**Reply by stockiNail** [reply]
Thank you very much @kurkle. I really appreciate!

**Reply by stockiNail** [reply]
anyway @kurkle take your time because I would need also for DataLabels but I see there is still a bug to solve I guess

**Reply by kurkle** [reply]
A lot of changes after beta1, so released beta2. Will give it some more time.

**Reply by stockiNail** [reply]
I'm gonna test beta2 this afternoon

**Comment by stockiNail** [comment, marked as answer]
Release 1.0.0 has been released! Thanks a lot! https://github.com/chartjs/chartjs-plugin-zoom/discussions/520
